db/migrate/20230528001749_change_diaper_pickup_person_to_pick_up_person.rb

```diff
@@ -1,24 +1,20 @@
 class ChangeDiaperPickupPersonToPickUpPerson < ActiveRecord::Migration[7.0]
   def up
-    [Organization, Partner].each do |model|
-      model.find_each do |record|
-        if record.partner_form_fields.include? 'diaper_pickup_person'
-          record.partner_form_fields.delete('diaper_pickup_person')
-          record.partner_form_fields << 'pick_up_person'
-          record.save
-        end
+    Organization.find_each do |organization|
+      if organization.partner_form_fields.include? 'diaper_pickup_person'
+        organization.partner_form_fields.delete('diaper_pickup_person')
+        organization.partner_form_fields << 'pick_up_person'
+        organization.save
```

Comment:
Close -- but as you have it,  this is going to have a side effect of reordering the edit and show screens.   Can you change the value in-place?